js/modules/loki_snode_api.js

```diff
@@ -42,8 +240,10 @@ class LokiSnodeAPI {
         let timeoutTimer = null;
         // private retry container
         const trySeedNode = async (consecutiveErrors = 0) => {
+
+          // Removed limit until there is a way to get snode info
+          // for individual nodes (needed for guard nodes)
```

Comment:
if no limit, do we expect to get all the snodes records? if so, might be worth noting

Reply:
yes, udapted